src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -68,6 +69,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case HelpCommand.COMMAND_WORD:
             return new HelpCommand();
 
+        case CreateFolderCommand.COMMAND_WORD:
+            return new CreateFolderCommandParser().parse(arguments);
+
```

Comment:
perhaps `AddressBookParserTest.java` should have a test case corresponding to the `CreateFolderCommand`?

Reply:
I agree. I've added the relevant test!